internal/configurations/configurations.go

```diff
@@ -234,61 +234,85 @@ func (c *Configuration) GetSecret(ctx context.Context) (*v1.Secret, error) {
 }
 
 // ForService returns a slice of Secrets matching the given Service.
-func ForService(ctx context.Context, kubeClient *kubernetes.Cluster, namespace, name string) ([]v1.Secret, error) {
+func ForService(ctx context.Context, kubeClient *kubernetes.Cluster, service *models.Service) ([]v1.Secret, error) {
 	secretSelector := labels.Set(map[string]string{
-		"app.kubernetes.io/managed-by": "Helm",
-		"app.kubernetes.io/instance":   names.ServiceHelmChartName(name, namespace),
-		ConfigurationLabelKey:          "true",
-		ConfigurationTypeLabelKey:      "service",
+		"app.kubernetes.io/instance": names.ServiceHelmChartName(service.Meta.Name, service.Meta.Namespace),
```

Comment:
Crossplane sets this label to the correct value on the Secrets, right?

Reply:
Yes, you can add any label or annotations on the secrets, so you just need to add them in your Epinio Service:

https://github.com/enrichman/helm-charts/blob/75093b8734bb6c50ff2f13e918038ca56ceb169e/charts/dynamodb-test/templates/dynamodbtable.yaml#L9-L13